src/validation.cpp

```diff
@@ -2456,6 +2464,7 @@ static void UpdateTip(CTxMemPool& mempool, const CBlockIndex* pindexNew, const C
             WarningBitsConditionChecker checker(bit);
             ThresholdState state = checker.GetStateFor(pindex, chainParams.GetConsensus(), warningcache[bit]);
             if (state == ThresholdState::ACTIVE || state == ThresholdState::LOCKED_IN) {
+                // DELAYED unreachable since min_lock_in_time = 0
```

Comment:
Because WarningBitsConditionChecker doesn't override MinLockInTime() so uses the default from AbstractThresholdConditionChecker which always returns 0. Will change the comment to refer to MinLockInTime instead of min_lock_in_time.